sphinx/source/_templates/layout.html

```diff
@@ -0,0 +1,154 @@
+{%- extends "basic/layout.html" %}
```

Comment:
Instead of copying/adapting the full template of pydata_sphinx_theme, I *think* you should be able to do `{% extends "pydata_sphinx_theme/layout.html" %}`, and then only override the blocks where you deviate

Reply:
@jorisvandenbossche looks like can definitely trip out some, but I don't see any sub-blocks in `content` that look like good hooks to put our scripts, etc so still hand-copying all of that (big) block. Mayb you can add an empry `extracontent` sub-block for users to purely add new bits to?